### PR TITLE
Bump the host compiler to 2.082.1

### PR DESCRIPTION
**Pull request by wilzbach**
Maybe https://github.com/dlang/dub/pull/1513 (reimplementation of the dependency resolution) helps to fix the current failure.

### CODE REVIEW

**Comment by wilzbach** [comment]
CC @CyberShadow

**Comment by CyberShadow** [comment]
```
curl: (22) The requested URL returned error: 403 Forbidden
```
What happened there?

**Comment by thewilsonator** [comment]
The title and commit message are wrong: the host compiler is already 2.081.1, its being upgraded to 2.083.0-beta.2

File: posix.mak [hunk 1]
```diff
@@ -180,7 +180,7 @@ DRUNTIME_LATEST_DIR=$G/druntime-${LATEST}
 PHOBOS_LATEST_DIR=$G/phobos-${LATEST}
 
 # stable dub and dmd versions used to build dpl-docs
-STABLE_DMD_VER=2.081.1
+STABLE_DMD_VER=2.083.0-beta.2
 STABLE_DMD_ROOT=$(GENERATED)/stable_dmd-$(STABLE_DMD_VER)
 STABLE_DMD_URL=http://downloads.dlang.org/releases/2.x/$(STABLE_DMD_VER)/dmd.$(STABLE_DMD_VER).$(OS).zip
```
[the remark this answers is not in the document]
Comment (wilzbach) [reply posix.mak]: Thanks I actually submitted the PR from my phone as I was travelling yesterday and I thought that this trivial version bump might do the trick.
In any case, it's even a bit trickier as we also need to add "-pre" (the URL is for now http://downloads.dlang.org/pre-releases)

**Comment by CyberShadow** [comment]
Thanks. Optimistically enabling auto-merge.

**Comment by wilzbach** [comment]
```
 .generated/stable_dmd-2.083.0-beta.2/dmd2/linux/bin64/dub build --compiler=.generated/stable_dmd-2.083.0-beta.2/dmd2/linux/bin64/dmd --root=ddoc && \
	mv ddoc/ddoc_preprocessor .generated/ddoc_preprocessor
Fetching libdparse 0.8.7 (getting selected version)...
Fetching stdx-allocator 2.77.2 (getting selected version)...
Invalid source/import path: /dev/shm/dtest/work/repo/dmd/generated/dub
Running pre-generate commands for dmd:frontend...
Performing "debug" build using .generated/stable_dmd-2.083.0-beta.2/dmd2/linux/bin64/dmd for x86_64.
dmd:frontend ~master: building configuration "library"...
stdx-allocator 2.77.2: building configuration "library"...
libdparse 0.8.7: building configuration "library"...
ddoc_preprocessor ~master: building configuration "executable"...
ddoc/source/assert_writeln_magic.d(202,5): Deprecation: The `delete` keyword has been deprecated.  Use `object.destroy()` (and `core.memory.GC.free()` if applicable) instead.
Linking...
```

Looks good so far. The deprecation would get fixed by https://github.com/dlang/dlang.org/pull/2496

**Comment by MartinNowak** [comment]
Should have been fixed for stable as well :/.

**Pull request merged**